### ISSUE TITLE
Setting up a Blog engine

### ISSUE THREAD

**Issue by gabrielbdornas** [infovote-io/dashboard#18]
@danybonfil I think a blog engine could be a great tool to grow our dashboard audience. As explained [here](https://squidfunk.github.io/mkdocs-material/blog/2022/09/12/blog-support-just-landed/) material Mkdocs provide this functionality for [insiders](https://squidfunk.github.io/mkdocs-material/insiders/).

We should consider this possibility.

**Comment by danybonfil** [comment]
Yes I think we can include for the audience interested in the theory behind the dashboard

**Comment by gabrielbdornas** [comment]
@danybonfil as discussed, we'll implement the blog engine. That's why I removed the `question` flag out of this issue. Reading one more time the [Mkdocs material insiders' documentation](https://squidfunk.github.io/mkdocs-material/insiders/#12000-piri-piri), it is possible to see that [setting up a blog](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#monday-january-31-2022) is now available without payments. This is excellent news, isn't it?
